tests/baselines/reference/user/npm.log

```diff
@@ -752,11 +752,8 @@ node_modules/npm/lib/uninstall.js(27,24): error TS2339: Property 'config' does n
 node_modules/npm/lib/uninstall.js(31,21): error TS2339: Property 'config' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/uninstall.js(32,32): error TS2339: Property 'globalDir' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/uninstall.js(33,19): error TS2339: Property 'prefix' does not exist on type 'typeof EventEmitter'.
-node_modules/npm/lib/uninstall.js(40,42): error TS2339: Property 'run' does not exist on type 'Uninstaller'.
 node_modules/npm/lib/uninstall.js(43,31): error TS2339: Property 'localPrefix' does not exist on type 'typeof EventEmitter'.
-node_modules/npm/lib/uninstall.js(46,50): error TS2339: Property 'run' does not exist on type 'Uninstaller'.
-node_modules/npm/lib/uninstall.js(51,27): error TS2507: Type 'typeof Installer' is not a constructor function type.
-node_modules/npm/lib/uninstall.js(70,36): error TS2339: Property 'idealTree' does not exist on type 'Uninstaller'.
+node_modules/npm/lib/uninstall.js(69,11): error TS2340: Only public and protected methods of the base class are accessible via the 'super' keyword.
```

Comment:
good: classes can now extend from constructor functions (HA HA. I thought nobody did this, but of course npm does.)
bad: resolution of `super.method` incorrectly thinks that prototype method assignments are not public. (they are since we don't support `@private`)
ugly: This only repros with `target: "es5"`. And also, the containing class is peak OO: `class Uninstaller extends Installer`.

Filed #26475